_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -240,126 +243,57 @@ This [HTML document][] has two [keyboard shortcuts][keyboard shortcut] using onl
 
 #### Passed Example 7
 
-This [HTML document][] has a single [printable character][] [keyboard shortcut][] for a [event target][] (using the `+` character), which is only available when that [event target][] has [focus][].
-
-```html
-<html>
-  <head>
-    <title>Passed Example 7</title>
-    <script src="/test-assets/ffbc54/shortcut.js"></script>
-  </head>
-
-  <body onload="registerShortcut({shortcutKey: '+', focusOnly: true}); activateShortcuts();">
-    <label for="target">Add to list (press "+" to add):</label>
-    <input type="text" id="target" />
-    <br />
-    <div>
-      To do list
-    </div>
-    <ul id="list"></ul>
-  </body>
-</html>
-```
-
-### Failed
-
-#### Failed Example 1
-
-This [HTML document][] has a [keyboard shortcut][] using only a [printable character][] that cannot be disabled.
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, and:
+ - exists an [instrument][] to prevent any result of the [keyboard event][]; and
+ - exists an [instrument][] to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false`; and
```

Comment:
```suggestion
 - there exists an [instrument][] to prevent any result of the [keyboard event][]; and
 - there exists an [instrument][] to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false`; and
```